docs/resources/administration_org_user_group.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: "spotinst"
+page_title: "Spotinst: health_check"
+subcategory: "Elastigroup"
+description: |-
+  Provides a Spotinst Health Check resource.
+---
+
+# spotinst\_administration\_org\_user\_group
+
+Provides a Spotinst user-group of your Spot organization
+
+## Example Usage
+
+```hcl 
+resource "spotinst_administration_org_user_group" "terraform_user_group" {
+  name = "test_user_group"
+  description = "user group by terraform"
+  user_ids = ["u-372ef6ae"]
+  policies{
+    account_ids = ["act-9c46c6df"]
+    policy_id = "pol-v75d8c06"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) User group name.
```

Comment:
name is not a required param as per document

Reply:
it is required